JK Hera/src/main/java/ayds/hera/songinfo/home/view/SongDescriptionHelperImpl.kt

```diff
@@ -8,8 +8,8 @@ interface SongDescriptionHelper {
   fun getSongDescriptionText(song: Song = EmptySong): String
 }
 
-internal class SongDescriptionHelperImpl : SongDescriptionHelper {
-  override fun getSongDescriptionText(song: Song): String {
+internal class SongDescriptionHelperImpl(private val releaseDateFormatter: ReleaseDateFormatter) : SongDescriptionHelper {
```

Comment:
Cuando empiezan a tener lineas largas por parametros les conviene usar este formato

```suggestion
internal class SongDescriptionHelperImpl(
    private val releaseDateFormatter: ReleaseDateFormatter
): SongDescriptionHelper {
```

Reply:
Y la linea de abajo les quedo mal indentada no? Veo ese espacio verde agregado